enclave/chain_relay/src/justification.rs

```diff
@@ -141,16 +110,15 @@ impl<Block: BlockT> GrandpaJustification<Block> {
 		let mut buf = Vec::new();
 		let mut visited_hashes = HashSet::new();
 		for signed in self.commit.precommits.iter() {
-			if communication::check_message_sig_with_buffer::<Block>(
+			if !sp_finality_grandpa::check_message_signature_with_buffer(
 				&finality_grandpa::Message::Precommit(signed.precommit.clone()),
 				&signed.id,
 				&signed.signature,
 				self.round,
 				set_id,
 				&mut buf,
-			)
-			.is_err()
-			{
+			) {
+				debug!("Bad signature on message from {:?}", &signed.id);
```

Comment:
Shouldn't this be `warn!` or `error`?

Reply:
An error is returned, so it will be handled as an error! later on. I have added this debug message to match substrates std version: https://github.com/paritytech/substrate/blob/master/primitives/finality-grandpa/src/lib.rs#L369-L370

Reply:
Ofc we can change it if you think it makes sense?

Reply:
Ok that makes sense. NOw that I think about it, we don't learn much from this information anyhow. It should be fine